### PR TITLE
feat: update withdraw quote in background

### PR DESCRIPTION
**Pull request by depressedPlumber502**

### CODE REVIEW

File: src/features/machines/withdrawUIMachine.ts [hunk 1]
```diff
@@ -553,6 +553,11 @@ export const withdrawUIMachine = setup({
             "not prepared"
           )
 
+          const quote = context.preparationOutput.value.swap?.swapQuote
+          if (!quote) {
+            throw new Error("quote not available")
```
[the remark this answers is not in the document]
Comment (depressedPlumber502) [reply src/features/machines/withdrawUIMachine.ts:558]: I just copy pasted from other place basically

[the remark this answers is not in the document]
Comment (depressedPlumber502) [reply src/features/machines/withdrawUIMachine.ts:558]: will adjust

File: src/features/machines/swapIntentMachine.ts [hunk 2]
```diff
@@ -54,7 +54,7 @@ type IntentOperationParams =
   | {
       type: "withdraw"
       tokenOut: BaseTokenInfo
-      quote: AggregatedQuote | null
+      quote: AggregatedQuote
```
[the remark this answers is not in the document]
Comment (depressedPlumber502) [reply src/features/machines/swapIntentMachine.ts]: Adjusted ser

File: src/features/machines/withdrawUIMachine.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (depressedPlumber502) [reply src/features/machines/withdrawUIMachine.ts]: Removed this check at all

**Review by cawabunga-bytes — APPROVED** [review]
Looks good!

**Pull request merged**